Corona-Warn-App/build.gradle

```
@@ -264,7 +264,7 @@ dependencies {
     debugImplementation 'androidx.fragment:fragment-testing:1.2.4'
 
     // CoraLibre sdk including the FOSS EN service reimplementation
-    implementation 'CoraLibre:sdk'
+    implementation 'com.github.Stypox:CoraLibre-android-sdk:540e298ce78949281e71626a3e136b1d81b60645'
```

This targets the wrong project and an outdated tag.
```suggestion
    implementation 'com.github.CoraLibre:CoraLibre-android-sdk:4d4cc65a6ead6437863c5d4adad7e0aa67da54e4'
```